src/serializer/Referentializer.js

```diff
@@ -219,33 +218,17 @@ export class Referentializer {
     }
   }
 
-  referentialize(
-    unbound: Set<string>,
-    instances: Array<FunctionInstance>,
-    shouldReferentializeInstanceFn: FunctionInstance => boolean
-  ): void {
+  referentialize(unbound: Set<string>, instances: Array<FunctionInstance>): void {
     for (let instance of instances) {
       let residualBindings = instance.residualFunctionBindings;
 
       for (let name of unbound) {
         let residualBinding = residualBindings.get(name);
-        invariant(residualBinding !== undefined);
+        if (residualBinding === undefined) continue;
         if (residualBinding.modified) {
           // Initialize captured scope at function call instead of globally
           if (!residualBinding.declarativeEnvironmentRecord) residualBinding.referentialized = true;
           if (!residualBinding.referentialized) {
-            if (!shouldReferentializeInstanceFn(instance)) {
-              // TODO #989: Fix additional functions and referentialization
-              this.realm.handleError(
-                new CompilerDiagnostic(
-                  "Referentialization for prepacked functions unimplemented",
-                  instance.functionValue.loc,
-                  "PP1005",
```

Comment:
Normally I'd ask you to delete this from the Wiki, but I see that this error number is already used for something else altogether, so two wrongs make a right here.